packages/blade/src/components/Tooltip/_decisions/decisions.md

```diff
@@ -0,0 +1,107 @@
+# Tooltip API Decisions <!-- omit in toc -->
+
+The tooltip typically provides additional context about the element or its function. A tooltip is always triggered by a `mouse hover` on desktop and `on tap` on mobile.
+
+<img src="./tooltip-thumbnail.png" width="380" alt="" />
+
+- [Design](#design)
+- [`Tooltip` API](#tooltip-api)
+  - [Examples:](#examples)
+    - [Basic](#basic)
+- [Library](#library)
+- [Motion](#motion)
+- [Accessibility](#accessibility)
+- [Open Questions](#open-questions)
+
+## Design
+
+[Figma Link](https://www.figma.com/file/jubmQL9Z8V7881ayUD95ps/Blade---Payment-Light?type=design&node-id=40540-559304&t=tmTrf3xJU6oj59fM-0) to all variants of the Tooltip component
+
+## `Tooltip` API
+
+| Prop          | Type                                                                     | Default     | Description                                                               | Required |
+| ------------- | ------------------------------------------------------------------------ | ----------- | ------------------------------------------------------------------------- | -------- |
+| content       | `string`                                                                 | `undefined` | Content of the tooltip                                                    | ✅       |
```

Comment:
Do we've a usecase for ellipsizing the content if it's large?

Reply:
No we only have maxWidth the height can be increased as needed tho from design side there are guidelines to not make tooltip content too long.